### DISCUSSION TITLE
Blue screen error of Windows machine during SLEAP training

### DISCUSSION THREAD

**Discussion by skyshi0** [talmolab/sleap#2042] [Help!]
Hi, I’m encountering some issues of the SLEAP Usage of GPU. (1)I couldn't initialize a training with more frames labeled. (1209). I was able to train the model at the beginning, but later when I added more frames, there’s an error massage “An error occurred while training centroid. Your command line terminal may have more information "shown on the screen when I was training the multi-animal top down model and I couldn’t find the exact reason shown in the terminal. (2)My Windows computer will show this ‘Blue Screen Error’ if I try to initialize the training several times, and I observed that it may be related to the maximal usage of the GPU. I’ve decreased the batch size to 1 but it still happened. I installed SLEAP by using Mambaforge, and I attached my computer GPU/CUDA/CUDA Toolkit version here:
NVIDIA Driver version: 566.03
CUDA Version: 12.7
System installed CUDA Toolkit: v11.8(check nvcc –version in terminal)
SLEAP installed CUDA Toolkit: v11.3(nvcc –version in sleap environment)

Do you think the CUDA/CUDA Toolkit has any conflict in my computer, and do you have any suggestions to troubleshoot this problem?
![errors during training](https://github.com/user-attachments/assets/9dd8d6e7-bffd-4a39-bc0b-08224af48430)
![errors during training2](https://github.com/user-attachments/assets/8e491842-8354-478d-b36b-11ee74ce2962)
![errors during training3](https://github.com/user-attachments/assets/a30547d7-acf7-4529-9db1-949b35f2191e)
![errors during train
![31aeee7c95125eb768c027333d685b0](https://github.com/user-attachments/assets/ce198e1a-fca9-4a34-b9f8-e80a500415e3)
ing4](https://github.com/user-attachments/assets/2bfc9372-60a4-4490-8a44-c2bd14f3dd06)

**Comment by talmo** [comment]
Hi @skyshi0,

Do you have the console logs a few lines below your second screenshot? That's where we might see the error.

Thanks!

Talmo

**Reply by skyshi0** [reply]
Yes, I attached here. I didn't see any errors and it seemed that the centered instance model was continuing, so I hit the keyboard interrupt.
![errors during training5](https://github.com/user-attachments/assets/43c3a2b5-bcde-4586-b1fb-3bf349b1e256)
![errors during training6](https://github.com/user-attachments/assets/763a5f53-26e9-430c-93fe-8e8d558d97e5)

**Reply by skyshi0** [reply]
I attached another readout when I trained a single instance model, and the error indicated the resource exhausted
![errors during training7](https://github.com/user-attachments/assets/4515d974-a195-4e61-8826-682a465b8c5e)

**Comment by eberrigan** [comment]
Hello @skyshi0,

This seems similar to the discussion #1889. You can try disabling the auto crop option for centered instance. 

That tensor is very large. What is the resolution of your images?

Can you send us your training package by uploading it with your training images embedded to his [link](https://docs.google.com/forms/d/e/1FAIpQLSfWBwcuZH8lxGyFabMAODLjSUE4YrSCYbrrnYvXCpgBqZpcJA/viewform).

Best,

Elizabeth

**Reply by skyshi0** [reply]
Thank you! The video is 1440x1080px, and I've sent my training package with videos to the link.
